package/ci/travis.yml

```diff
@@ -98,13 +98,19 @@ matrix:
     env:
     - JOBID=ios
     - TARGET=ios-simulator
-  - language: cpp
-    os: osx
+  - language: generic
+    os: ubuntu
+    services:
+    - docker
     # Using whatever is the default, since that could be hopefully most
     # up-to-date
     env:
     - JOBID=emscripten
     - TARGET=emscripten
+    - EMSCRIPTEN_VERSION=2.0.7
```

Comment:
I want to test on the oldest version available, ideally exactly what was there on Mac (1.38.42)

Reply:
The docker image supports everything above 1.39.0, 1.38.42 is pretty ancient

Reply:
ok then 1.39.0 :)